### PR TITLE
#608 - Updated docs for maven proxy configuration

### LINKED ISSUE #608: Document how to configure maven-proxy with authentication
I would like to proxy private nexus/artifactory proxy. It is not evident to me how to configure such maven proxy from the current documentation/tests/samples.

### PR DESCRIPTION
**Pull request by olegmoz**
Closes #608 
Updated docs for maven proxy configuration

### CODE REVIEW

File: examples/maven/README.md [hunk 1]
```diff
@@ -68,26 +68,29 @@ Run `mvn install` (or `mvn install -U` to force download dependencies).
...
+Proxy repository supports caching in local storage.
+To enable it and make previously accessed images available when source repository is down 
+add `storage` section to config:
+
+```yaml
+repo:
+  type: maven-proxy
+  remotes:
+    - url: https://repo.maven.apache.org/maven2
+      storage:
```
Comment (olenagerasimova) [inline examples/maven/README.md:93]: @olegmoz are you sure it's correct level in yaml for "storage" section?

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@olegmoz one question, please, take a look

**Comment by olegmoz** [comment]
@olenagerasimova thank you very much for noticing the problem with `storage` in YAML, fixed it

**Review by olenagerasimova — APPROVED** [review]
@olegmoz thanks, good to merge

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#689` is not assigned, can't get performer

**Pull request merged**